front/app/components/admin/TabbedResource/index.tsx

```diff
@@ -109,6 +118,27 @@ function getRegularExpression(tabUrl: string) {
   return new RegExp(`^/([a-zA-Z]{2,3}(-[a-zA-Z]{2,3})?)(${tabUrl})(/)?$`);
 }
 
+const FormattedTabLink = ({
+  tab: { url, label, statusLabel },
+}: {
+  tab: ITab;
+}) => {
+  if (!isNilOrError(statusLabel)) {
```

Comment:
Seems a bit overkill to do `isNilOrError` when the type is `string | undefined`, maybe just `if (statusLabel)` would do here?

Reply:
My last codebase wasn't entirely typescript so I'm used to coding a bit more defensively than might be necessary 😅  I'll change it. Thanks for the review 🥳